electrumx/server/daemon.py

```diff
@@ -69,6 +70,15 @@ async def __aexit__(self, exc_type, exc_value, traceback):
     def connector(self):
         return None
 
+    def request_staking(self, params_str):
```

Comment:
misleading name, I would call it `request_stake`

Reply:
fixed